### PR TITLE
e.io Webhooks

### PR DESCRIPTION
**Pull request by pnedelko**

### CODE REVIEW

File: services/webhooks/spec/support/jasmine.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (anton-kotenko) [inline services/webhooks/spec/support/jasmine.json]: Are we able to finally remove jasmine?

File: services/webhooks/src/body-parser.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
+'use strict';
```
Comment (anton-kotenko) [inline services/webhooks/src/body-parser.js]: no use strict by codestyle

File: services/webhooks/src/request-handlers/post.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+const REQUEST_FIELDS = [
+    'headers',
+    'url',
+    'method',
+    'originalUrl',
+    'query',
+    'body'
+];
+
+// class Message {
```
Comment (anton-kotenko) [inline services/webhooks/src/request-handlers/post.js]: maybe typedef or doc?

File: services/webhooks/src/http-api.js [hunk 4]
```diff
@@ -0,0 +1,197 @@
+const express = require('express');
```
Comment (anton-kotenko) [inline services/webhooks/src/http-api.js]: Am i right that all this will be moved  into /src/XXX folder?

File: services/webhooks/.eslintrc.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (anton-kotenko) [inline services/webhooks/.eslintrc.json]: As far as i know this service should  in OIH is called `communication-router`. Am i right?

File: services/webhooks/README.md [hunk 6]
```diff
@@ -0,0 +1,2 @@
+# Webhooks
+Elastic.io webhooks service as is.
```
Comment (anton-kotenko) [inline services/webhooks/README.md]: Readme should be updated

File: services/webhooks/src/http-api.js [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+function errorHandler(err, req, res, next) { //eslint-disable-line no-unused-vars
+    res.status(err.statusCode || 500).json({
+        error: err.message
+    });
+}
+
+class Api {
+    constructor(config, flowsDao) {
+        this._flowsDao = flowsDao;
+        this._logger = logger;
```
Comment (anton-kotenko) [inline services/webhooks/src/http-api.js]: may be logger also should be injected as service through constructor?

File: services/webhooks/src/body-parser.js (hunk 2 above)
Comment (pnedelko) [reply services/webhooks/src/body-parser.js]: +

File: services/webhooks/src/request-handlers/post.js (hunk 3 above)
Comment (pnedelko) [reply services/webhooks/src/request-handlers/post.js]: removed for now

File: services/webhooks/spec/support/jasmine.json (hunk 1 above)
Comment (pnedelko) [reply services/webhooks/spec/support/jasmine.json]: not this time

File: services/webhooks/src/http-api.js (hunk 4 above)
Comment (pnedelko) [reply services/webhooks/src/http-api.js]: yes

**Comment by pnedelko** [comment]
Closing in favour of https://github.com/openintegrationhub/openintegrationhub/pull/156.

**Pull request closed without merging**